Translate student login and settings guide (#496)
dodona-edu/dodona-edu.github.io: en/guides/students/login-and-settings/index.md

---
title: "Signing Up & User Settings"
description: "Tutorial: signing up and user settings"
order: 2
---

# Signing Up & User Settings

> On this page, you will find all the information you need to create and personalize a user account on Dodona as a student.

Reviewer: ```suggestion
> On this page, you will find all the information you need to create and personalize a user account on Dodona as a student.
```

Is there a reason why the table of contents wasn't copied to the English version?

Author: Since the latest style change, tables of contest are always include on the right side of the page

## Selecting the Language

You can use Dodona in English or Dutch. Use the language menu on the right side of the navigation bar (at the top of every page) for this purpose.

![image](./choose_language.png)

If you select a language while logged in, it will also be set as your preferred language in your [user profile](https://dodona.be/profile). This way, your preferred language will be automatically selected upon login.

::: tip Is Everything Translated?

The fixed components of Dodona (menu bars, course structure, notifications, etc.) are always displayed in the selected language.

The name and description of a [course](../courses/) are displayed in the language as entered by the instructor. These are not adapted to your language preferences.

The name and description of an [exercise](../exercises/#navigating-to-an-exercise) are always displayed in your preferred language if available. Not all exercises are available in both languages.

:::

> ## Signing Up

Reviewer: sign up = registreren; sign in = aanmelden

Author: I know.  And `sign up` is not required for Dodona.
But this can be confusing to some users. Which is why I wrote the guide as 'if you want to sign up, click sign in'
but maybe that has made it even more confusing.

If you want I can also replace all remaining mentions of `sign up` with `sign in` and then add a notice: there is no need to sign up for Dodona, you cal immediately sign in and start using the platform.

Reviewer: I think the guide is clear enough on that? I you want to sign up, simply click the sign in button and Dodona wil automatically create an account for you.

To sign up for Dodona, either click on `Sign in` on the right side of the navigation bar or navigate to the landing page and click the `Sign in` button in the center of the page.

![image](./login.png)

> You can sign up for Dodona by selecting your educational institution on the sign-up page. Secondary school students can log in with their Smartschool, Office365, or G-Suite account. If this is your first time signing up for Dodona, a new user account will be automatically created based on the personal information provided by your external educational or research institution.

Reviewer: ```suggestion
You can sign in to Dodona by selecting your educational institution on the sign-in page. Secondary school students can log in with their Smartschool, Microsoft 365, or Google Workspace account. If this is your first time signing in to Dodona, a new user account will be automatically created based on the personal information provided by your external educational or research institution.
```
This might need a change in Dutch as well.

![image](./sign_in.png)

::: warning Your Data on Dodona

> Some educational or research institutions explicitly ask for your consent to share some [personal data](https://dodona.be/en/data/) with Dodona at the end of the sign-up procedure. Often, you only need to give this consent once on each computer you use to sign up. Dodona uses this personal data solely for internal management of your user profile and never shares this information with external parties.

Reviewer: ```suggestion
Some educational or research institutions explicitly ask for your consent to share some [personal data](https://dodona.be/en/data/) with Dodona during the sign in procedure. Often, you only need to give this consent once on each computer you use to sign up. Dodona uses this personal data solely for internal management of your user profile and never shares this information with external parties.
```

:::

::: tip Error Message?

If no one from your school has signed up for Dodona before, things might not go as expected. To sign up, we use the existing accounts from your educational institution and link them using techniques such as [SAML](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [OAuth](https://en.wikipedia.org/wiki/OAuth). Sometimes, this process fails because your school does not grant permission for this link. Ask an ICT administrator or instructor from your educational or research institution to [contact us](../getting-started/#contact-us) if you want to give all user accounts from your institution access to Dodona.
:::

::: tip Multiple Accounts?

> If you have user accounts with different educational or research institutions, each corresponds to a separate user account on Dodona. Always use the same account to sign up.

Reviewer: ```suggestion
If you have user accounts with different educational or research institutions, each corresponds to a separate user account on Dodona. Always use the same account to sign in.
```

:::

Once you are logged in, the neutral landing page transforms into a personalized homepage with an overview of all [courses](../courses/) you are registered for, their exercise series with upcoming deadlines, and your progress in submitting solutions for [exercises](../exercises/). You can always return to the landing page by clicking on `Dodona` at the top left of any page.

If this is your first time signing up for Dodona, you are obviously not registered for any courses yet and have not submitted any solutions. Therefore, your homepage looks quite empty at the moment. You can find out how to register for a course on [this page](../courses/#registering-for-a-course).

## Setting Your Personal Preferences

Select `My Profile` in the user menu on the right side of the navigation bar to navigate to your profile page.

On the profile page, you will find your personal information (first name, last name, username, and email address) and some personal preferences ([language](#selecting-the-language) and time zone) that together form your user profile.

![image](./student.user_menu_my_profile.png)

> Click the edit button in the bottom right corner of your profile page to edit your user profile. You cannot modify your personal information such as your name and email address. These are automatically updated based on the data provided by your external educational or research institution during sign-up.

Reviewer: ```suggestion
Click the edit button in the bottom right corner of your profile page to edit your user profile. You cannot modify your personal information such as your name and email address. These are automatically updated based on the data provided by your external educational or research institution when signing in.
```
